docs/guides/kds-sync-name.md

```diff
@@ -0,0 +1,186 @@
+# What is the name of the resource after KDS sync?
```

Comment:
If the code for this is contained in one place in the code (which it should probably) we should just have this as comments in the code.